Source/Scene/GroundPrimitive.js

```diff
@@ -240,7 +240,7 @@ define([
 
         var readOnlyAttributes;
         if (defined(this.geometryInstances) && isArray(this.geometryInstances) && this.geometryInstances.length > 1) {
-            readOnlyAttributes = readOnlyInstanceAttributesScratch;
```

Comment:
This is to prevent every GroundPrimitive from creating the same array. It's not used here, but in the Primitive that it creates.